### ISSUE TITLE
Blank 'Torrents' folder prevents prevents 'Add Torrent To' icon in right-click list.

### ISSUE THREAD

**Issue by WhiteWolf-One** [GChristensen/torrent-add#3]
Win 10, qbitttorrent, currently 'Brave' (https://brave.com/): 

Looking to have downloads land in root folder, in my case, default of V:\

Blank 'Torrents' folder prevents 'Add Torrent To' icon in right-click list.

Leaving default of [root] sends downloads to V:\[root]

Currently attempting '.\'.

**Comment by GChristensen** [comment]
Tested on brave.

**Comment by GChristensen** [comment]
What I have tried:

1. Added a torrent to the root folder. It have appeared in the [root] subfolder. This happens only on qBittorrent, will be fixed in 0.2.6.
2. Removed [root] completely from the list of the categories. The context menu appears.
3. Removed [root] completely from the list of the categories, leaving leading colon. The context menu appears.
4. The content of the root folder should not influence the availability of the menu, since WebExtensions has no access to the filesystem.
